internal/proto/local/controller/api/services/v1/host_catalog_service.proto

```diff
@@ -103,6 +103,9 @@ message ListHostCatalogsResponse {
 
 message CreateHostCatalogRequest {
   api.resources.hostcatalogs.v1.HostCatalog item = 1;
+  // As an alternative to providing the plugin id in the provided HostCatalog,
```

Comment:
Could we just look at the `name` field in the host catalog iff id is empty?

Reply:
Then how would we set the name of the host catalog to something different from the plugin?

Reply:
Ah, sorry, right.